docs/docusaurus/multitenancy/tenant-migration.md

```diff
@@ -0,0 +1,176 @@
+# Migration from `apikey` to `JWT` authentication
+
+PRISM Agent authentication supports multiple authentication methods simultaneously.
+This means the user can seamlessly use any available credentials including `apikey` or `JWT` to access the wallet.
+The agent's UMA permission resource also exposes self-service permission endpoint, allowing users to manage the permissions for their wallets.
```

Comment:
We should introduce the acronym UMA to the reader?

Reply:
Agree with David, this needs defined and explained.

Reply:
Added reference to the glossary